pom.xml

```diff
@@ -19,7 +19,7 @@
         <sonar.coverage.jacoco.xmlReportPaths>../multiapps-coverage/target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
         <junit.version>5.9.2</junit.version>
         <junit-platform.version>1.9.2</junit-platform.version>
-        <mockito.version>4.11.0</mockito.version>
+        <mockito.version>5.10.0</mockito.version>
```

Comment:
Have you run the unit tests with this change? Since this is a major version upgrade it could lead to failing tests.

Reply:
Yes, I have. They pass